### PR TITLE
[EXPLORER_OLD] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: modules/rosapps/applications/explorer-old/explorer-ro.rc [hunk 1]
```diff
@@ -282,7 +282,7 @@ BEGIN
     IDS_NETWORK "Rețea"
     IDS_CONNECTIONS "Conexuni în rețea"
     IDS_DRIVES "Unități"
-    IDS_SEARCH_COMPUTER "Calculator…"
+    IDS_SEARCH_COMPUTER "computer…"
```
Comment (binarymaster) [inline modules/rosapps/applications/explorer-old/explorer-ro.rc:285]: Missing a word? en-US has `Search Computer...` here:

https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=IDS_SEARCH_COMPUTER

Comment (AndreiMiloiu) [reply modules/rosapps/applications/explorer-old/explorer-ro.rc]: Fixed!

**Comment by AndreiMiloiu** [comment]
Any news?

**Pull request merged**